pom.xml

```diff
@@ -113,6 +114,9 @@
     <!-- Filled in by "maven-hpi-plugin" with the path to "org-netbeans-insane-hook.jar" extracted from "jenkins-test-harness" -->
     <jenkins.insaneHook />
 
+    <!-- Filled in by "maven-hpi-plugin" with "-javaagent:/path/to/mockito-core-<version>.jar" -->
+    <jenkins.javaAgent />
```

Comment:
Will make the failure mode softer in the future by simply skipping a Java agent in the case that, for whatever reason, `maven-hpi-plugin` doesn't fill in this value.